### PR TITLE
Add more tests and clean up some code

### PR DESCRIPTION
**Pull request by LI-ayamanaka**

### CODE REVIEW

**Comment by jrusnakli** [comment]
This is failing checks on the one test (do to back button event not supported I think).

________________________________

File: orchestrator/src/androidtestorchestrator/device.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 class Device(object):
     """
-    Class for interacting directly with a remote device over USB via Google's adb command.
+    Class for interacting directly with a remote device over USB via Google's adb command, or with an emulator.
```
Comment (li-wjohnson) [inline orchestrator/src/androidtestorchestrator/device.py]: Maybe "...directly with a remote device over USB or local emulator..." 
or just "interacting with a device via Google's adb command."

File: orchestrator/test/test_application_class.py [hunk 2]
```diff
@@ -114,14 +116,36 @@ def test_monkey(self, device, support_app):  # noqa
...
             Application.from_apk("no.such.package", device)
+
+    def test_app_uninstall_logs_error(self, device: Device):
+        with patch("androidtestorchestrator.application.log") as mock_logger:
+            app = Application(package_name="com.android.providers.calendar", device=device)
+            app.uninstall()
+            assert mock_logger.error.called
+
+    def test_clean_kill(self, device: Device, support_app: str):
+        app = Application.from_apk(support_app, device)
```
Comment (li-wjohnson) [inline orchestrator/test/test_application_class.py]: Possibly could move the "install app; try: finally uninstall" bit into a fixture.

Comment (LI-ayamanaka) [reply orchestrator/test/test_application_class.py]: I can address that in a future commit. Want to get this one in for now.

Comment (li-wjohnson) [reply orchestrator/test/test_application_class.py]: Yup, agreed.

**Pull request merged**